apps/entropy-tester/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "@pythnetwork/entropy-tester",
+  "version": "1.0.0",
+  "description": "Utility to test entropy provider callbacks",
+  "main": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "files": [
+    "lib/**/*"
+  ],
+  "scripts": {
+    "build": "tsc",
+    "fix:format": "prettier --write \"src/**/*.ts\"",
+    "fix:lint": "eslint src/ --fix --max-warnings 0",
+    "test:format": "prettier --check \"src/**/*.ts\"",
+    "test:lint": "eslint src/ --max-warnings 0",
+    "start": "node lib/index.js",
+    "dev": "ts-node src/index.ts",
+    "prepublishOnly": "pnpm run build && pnpm run test:lint",
+    "preversion": "pnpm run test:lint",
+    "version": "pnpm run test:format && pnpm run test:lint && git add -A src"
+  },
```

Comment:
I suggest modeling your scripts after another app, e.g. https://github.com/pyth-network/pyth-crosschain/blob/main/apps/entropy-explorer/package.json#L9-L21 -- many of the scripts here don't make any sense for a package that isn't published like this and you're missing things like having prettier format all valid file types